moduleframework/tests/static/helpmd_lint.py

```diff
@@ -47,31 +47,38 @@ def setUp(self):
     def tearDown(self, *args, **kwargs):
         pass
 
-    def test_helpmd_exists(self):
-        self.assertTrue(self.helpmd)
+    def _get_msg(self, msg):
+        return msg + " is missing in help file."
 
     def test_helpmd_image_name(self):
         container_name = self.dp.get_docker_specific_env("NAME=")
         if container_name:
-            self.assertTrue(self.helpmd.get_image_name(container_name[0].split('=')[1]))
+            self.assertTrue(self.helpmd.get_image_name(container_name[0].split('=')[1]),
+                            msg="%s . Format is like %% MEMCACHED(1)" % self._get_msg("Image name"))
 
     def test_helpmd_maintainer_name(self):
         maintainer_name = self.dp.get_specific_label("maintainer")
         if maintainer_name:
-            self.assertTrue(self.helpmd.get_maintainer_name(maintainer_name[0]))
+            self.assertTrue(self.helpmd.get_maintainer_name(maintainer_name[0]),
+                            msg="%s Format is like '%% User Name'. Or you have a typo in help file." % self._get_msg("maintainer"))
```

Comment:
"Format is like" - how about "The correct format is"
Also s/typo in help file/typo in the help file/